debian/authd-example-broker.install

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/dh-exec
+
+usr/bin/examplebroker-bin => ${env:AUTHD_DAEMONS_PATH}/authd-examplebroker
+
+examplebroker/com.ubuntu.authd.ExampleBroker.conf /usr/share/dbus-1/system.d
+examplebroker/com.ubuntu.authd.ExampleBroker.service /usr/share/dbus-1/system-services
```

Comment:
I think this is still not what we want. IMHO, the examplebroker should be an executable that, when executed, exports the broker on `dbus` and cleans it up when the program finishes running. This way, we don't need to worry about having to install/uninstall it. Its purpose is to test `authd`, not the broker behaviour, so we don't need anything directly configured on `dbus` (as `authd` does not directly care about it).

Reply:
I see the point, but that wouldn't allow proper testing, especially if we want to do proper autopkgtests were you just install everything and then you run a client that would work as the system would be configured properly.

> so we don't need anything directly configured on dbus (as authd does not directly care about it).

In fact `authd` does care about it, because if that would happen when we run the daemon, then we wouldn't be aware of testing the case that authd does dbus-activation of the service when that you select the broker on the brokers list.

Also, not having a script like this would imply that we rely on the autopkgtest script to do that, but IMHO it's not nice to maintain the same in two different places.

What we may want to do, in case, is to make the examplebrorker to do the conf setup in case that's not set, and I can do that, but I feel we should still provide a way to create what would be the real setup in a properly configured machine.

Reply:
Ah, also having those files installed doesn't allow anything but just having the service to own the name, and while these may be handled by the service itself, as I said, I'd prefer to be able to do dbus-activation (but only through systemd that it's not installed by default).